internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -76,14 +76,6 @@ private ConfigKeys() {
    */
   public static final String KAFKA_LOG_COMPACTION_FOR_HYBRID_STORES = "kafka.log.compaction.for.hybrid.stores";
 
-  /**
-   * Whether to turn on Kafka's log compaction for the store-version topics of incremental push stores.
-   *
-   * Will take effect at topic creation time, and when the incremental push config for the store is turned on.
-   */
-  public static final String KAFKA_LOG_COMPACTION_FOR_INCREMENTAL_PUSH_STORES =
-      "kafka.log.compaction.for.incremental.push.stores";
```

Comment:
Hmm I am not sure if we should take this off...?

Reply:
Why not? We'll use hybrid store settings